extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -126,22 +122,19 @@ ConsumeWindowsEventLog::ConsumeWindowsEventLog(const std::string& name, utils::I
 }
 
 ConsumeWindowsEventLog::~ConsumeWindowsEventLog() {
-  if (xmlDoc_) {
-    xmlDoc_.Release();
-  }
-  CoUninitialize();
 }
 
 void ConsumeWindowsEventLog::initialize() {
   //! Set the supported properties
-  setSupportedProperties({Channel, Query, RenderFormatXML, MaxBufferSize, InactiveDurationToReconnect, IdentifierMatcher, IdentifierFunction });
+  setSupportedProperties({Channel, Query, MaxBufferSize, InactiveDurationToReconnect, IdentifierMatcher, IdentifierFunction, ResolveAsAttributes });
 
   //! Set the supported relationships
   setSupportedRelationships({Success});
 }
 
 void ConsumeWindowsEventLog::onSchedule(const std::shared_ptr<core::ProcessContext> &context, const std::shared_ptr<core::ProcessSessionFactory> &sessionFactory) {
 	context->getProperty(IdentifierMatcher.getName(), regex_);
+	context->getProperty(ResolveAsAttributes.getName(), resolve_as_attributes_);
```

Comment:
Doesn't belong to the scope of this PR, but I this makes me raise a concern: as the property is not required, what happens in case it's removed and the processor is restarted? As far as I understand we are not going to override the value.